app/serializers/api/v1/ecf_user_serializer.rb

```diff
@@ -25,10 +25,15 @@ class ECFUserSerializer
       }.freeze
 
       set_id :id
-      attributes :email, :full_name
+      attributes :full_name
+
+      attributes :email do |user|
+        preferred_identity = find_current_induction_record(user)&.preferred_identity
+        preferred_identity&.email || user.email
+      end
 
       attributes :user_type do |user|
-        case user.teacher_profile.ecf_profiles.last&.type
+        case find_ecf_profile(user)&.type
```

Comment:
Really minor, but these `attributes` can be singular as they're defining just one attribute.

Reply:
These have now been removed as I have moved to an induction_record focused serialiser